themes/edx.org/lms/templates/dashboard.html

```diff
@@ -141,10 +141,10 @@ <h2 class="header-courses">${_("My Courses")}</h2>
         <label for="dashboard-search-input">${_('Search Your Courses')}</label>
         <div class="search-field-wrapper">
           <input id="dashboard-search-input" type="text" class="search-field"/>
-          <button type="submit" class="search-button" aria-label="${_('Search')}">
+          <button type="submit" class="search-button" title="${_('Search')}">
             <i class="icon fa fa-search" aria-hidden="true"></i>
```

Comment:
Yeah I'm not seeing these in that file. Let me pull the branch and see if they show up.